Create Purpose of MS Team Channels.md

mohit23iitk opened the pull request.

mohit23iitk committed:
Create Purpose of MS Team Channels.md

jjalan commented on product-management/Purpose of MS Team Channels.md:
nit: name file case insensitive, without spaces. ex: `ms-teams-channel.`

jjalan commented on product-management/Purpose of MS Team Channels.md:
```diff
@@ -0,0 +1,140 @@
+# Overview
```
The content in this doc is largely accurate, its too long. The problem with that is people would tend to overlook it. We should try to say the same but in 20% of words used in this doc, without leaving key details.

mohit23iitk committed:
ms-team.md

Updated the File Name. 

Updated the content to be more precise and less verbose.

jjalan reviewed (commented):
Can you check few other pages? I think a header with layout is added so the pages works well with Github pages. See this [PR](https://github.com/jalantechnologies/handbook/pull/47) for details.

mohit23iitk committed:
Update ms-team.md
